client/signup/steps/import-url-onboarding/index.jsx

```diff
@@ -107,7 +107,7 @@ class ImportURLOnboardingStepComponent extends Component {
 
 	handleInputRef = ( el ) => ( this.inputRef = el );
 
-	focusInput = () => invoke( this.inputRef, 'focus' );
+	focusInput = () => this.inputRef.focus();
```

Comment:
There should be a `?.focus()` check here. The `input` element is not necessarily rendered and the ref attached -- `renderFallbackEngines()` can be done instead. But `focusInput()` is called anyway in `componentDidMount`.

Other usages of `invoke` should be also checked, but except this one, they look safe to me.

Reply:
👍 I think I'll add safeguards to them all, just in case. All of them look as if they could eventually be more fragile than before.

Reply:
Done in 679424035965434e4772973589a942f8bac8fbcf.

Reply:
If you don't mind a drive-by change that's not 100% Lodash focused, it should be straighforward to refactor these refs to `React.createRef` .

Reply:
I prefer leaving this for another time if you don't mind. There are a bunch more of those we could (and should refactor).